demos/sveltekit/vite.config.ts

```diff
@@ -1,9 +1,10 @@
 import { sentrySvelteKit } from '@sentry/sveltekit';
 import { sveltekit } from '@sveltejs/kit/vite';
 import { defineConfig } from 'vitest/config';
+import spotlight from '@spotlightjs/spotlight/vite-plugin';
 
 export default defineConfig({
-	plugins: [sveltekit(), sentrySvelteKit({ autoUploadSourceMaps: false })],
+	plugins: [sveltekit(), sentrySvelteKit({ autoUploadSourceMaps: false }), spotlight()],
```

Comment:
@dcramer this is how you use the plugin. Just import and enable for any project. For Astro, we still need `@spotlightjs/astro` for the tighter integration.